lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -928,6 +928,47 @@ void mlir::torch::onnx_c::populateDefaultDomainAtoF(
                       binder.op, resultType, operand);
                   return success();
                 });
+  patterns.onOp("Expand", 1,
+         [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+          // uses ideas and code from onnx.Reshape
+           Torch::ValueTensorType resultType;
+           Value data, shape;
+           if (binder.tensorOperands(data, shape) || binder.tensorResultType(resultType))
+             return failure();
+           Torch::BaseTensorType shapeType = shape.getType().cast<Torch::BaseTensorType>(); 
+           SmallVector<int64_t> selectSizes;
+           selectSizes.push_back(1);
+           Type selectResultType = shapeType.getWithSizesAndDtype(
+             llvm::ArrayRef(selectSizes), shapeType.getOptionalDtype());
```

Comment:
You can directly do:
```
Type selectResultType = shapeType.getWithSizesAndDtype({1}, shapeType.getOptionalDtype());
```

Reply:
So, If I change to passing {1} instead of llvm::ArrayRef(selectSizes), I get torch.aten.select.int %arg1, %int0, %int1 : !torch.vtensor<[2],si32>, !torch.int, !torch.int -> !torch.vtensor<[94349894303864],si32> instead of desired torch.aten.select.int %arg1, %int0, %int0_0 : !torch.vtensor<[2],si32>, !torch.int, !torch.int -> !torch.vtensor<[1],si32>. See the [94349894303864] instead of desired [1]. So, passing an implicit std::initializer_list in place of llvm::ArrayRef< T > to Type BaseTensorType::getWithSizesAndDtype(
    std::optional<ArrayRef<int64_t>> optionalSizes, Type optionalDtype) const { .. } is making it go nuts :-) Keep this one as is then?

Reply:
That's strange. Let's keep it the same then.